test/TesterAzureUtils/Streaming/DelayedQueueRebalancingTests.cs

```diff
@@ -72,13 +75,12 @@ public DelayedQueueRebalancingTests()
             await ValidateAgentsState(2, 4, "2");
         }
 
-        [SkippableFact(Skip = "https://github.com/dotnet/orleans/issues/3993"), TestCategory("Functional")]
+        [SkippableFact, TestCategory("Functional")]
```

Comment:
Shouldn't you remove the `SkippableFact` par entirely?

Reply:
No this test depends on AzureQueue to be set up, so it will still be skipped in jekins